lib/licensee/project_file.rb

```diff
@@ -12,7 +12,10 @@ class File
 
       def initialize(content, filename = nil)
         @content = content
-        @content.encode!(ENCODING, ENCODING_OPTIONS)
+        @content.force_encoding(ENCODING)
+        unless @content.valid_encoding?
```

Comment:
This is pretty good. Do note that encoding an UTF8-tagged script into UTF-8 with `:replace` will remove all the invalid characters altogether from the License -- as opposed to to Charlock's approach that actually figures out the encoding of those characters and transcodes them into valid UTF8...

So we're gonna be losing data here, and hence reducing accuracy. But I think this is an acceptable tradeoff, particularly since the text of all our supported licenses is usually ASCII English and this approach will only strip away things like author names.